Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM ubuntu
+FROM python:3.9.0-buster
```

Comment:
I'm a little bit included to stick to a plain ubuntu at this moment since I am concerned that the above buster may include too many packages, esp. if still we'll do apt and pip later in this file?